plugins/ingest-attachment/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -37,7 +37,10 @@ grant {
 
   // TODO: fix PDFBox not to actually install bouncy castle like this
   permission java.security.SecurityPermission "putProviderProperty.BC";
-  permission java.security.SecurityPermission "insertProvider";
+  permission java.security.SecurityPermission "insertProvider.BC";
+  permission java.security.SecurityPermission "removeProviderProperty.BC";
+  permission java.util.PropertyPermission "jdk.tls.rejectClientInitiatedRenegotiation", "write";
```

Comment:
Curious where `jdk.tls.rejectClientInitiatedRenegotiation` is coming from?

Reply:
it is cumming from BC same as for `permission java.security.SecurityPermission "removeProviderProperty.BC` found out it here: https://github.com/opensearch-project/security/pull/2901 when tested it for the security plugin